Assets/MRTK/Core/Providers/Hands/ArticulatedHandDefinition.cs

```diff
@@ -72,6 +78,37 @@ public bool IsInPointingPose
             }
         }
 
+        /// <summary>
+        /// Calculates whether the current the current joint pose is selecting (air tap gesture).
+        /// </summary>
+        public bool IsPinching
+        {
+            get
+            {
+                MixedRealityPose thumbTip;
+                MixedRealityPose indexTip;
+                if (unityJointPoses.TryGetValue(TrackedHandJoint.ThumbTip, out thumbTip) && unityJointPoses.TryGetValue(TrackedHandJoint.IndexTip, out indexTip))
+                {
+                    float distance = Vector3.Distance(thumbTip.Position, indexTip.Position);
```

Comment:
I think for distances using sqrdMagnitude is generally faster, though the speed increase depends on the platform:
https://docs.unity3d.com/ScriptReference/Vector3-sqrMagnitude.html
https://answers.unity.com/questions/125882/vectordistance-performance.html

Reply:
However at the same time, some of that guidance was written some time ago so I'm not actually super sure how much of that is relevant. Supposedly on mobile platforms it's still pretty noticable

Reply:
Leap Motion does not support mobile platforms, so I think I will keep it the way it is for now.